components/dropbox/actions/create-or-append-to-a-text-file/create-or-append-to-a-text-file.mjs

```diff
@@ -0,0 +1,87 @@
+import dropbox from "../../dropbox.app.mjs";
+import common from "../common.mjs";
+
+export default {
+  name: "Create or Append to a Text File",
+  description: "Adds a new line to an existing text file, or creates a file if it doesn't exist. [See docs here](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesUpload__anchor)",
+  key: "dropbox-create-or-append-to-a-text-file",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    dropbox,
+    name: {
+      type: "string",
+      label: "File name",
+      description: "Your new file name",
+    },
+    path: {
+      propDefinition: [
+        dropbox,
+        "pathFolder",
+      ],
+      optional: true,
+      description: "The file path in the user's Dropbox to create the folder. If not filled, it will be created in the root folder.",
```

Comment:
```suggestion
      description: "The file path in the user's Dropbox to create the file. If not filled, it will be created in the root folder.",
```